### ISSUE TITLE
What amp is this sim based on?

### ISSUE THREAD

**Issue by cyrusadkisson** [brummer10/GxVmk2.lv2#2]
You'd told me it was a Vox mkII a long time ago, but I can't find that model when I google. 

Are you sure it's not the Vox VXII? https://youtu.be/7CmGxx2McCs?t=34

Edit: Also, does this amp have no pre-gain? The "volume" knob is a master out control?

**Comment by brummer10** [comment]
Yes, I'm sure it's a MKII. I do it on request of a owner from one, he send me the schematic I used to create the plugin. 
Duno were I've it floating around right now.

**Comment by brummer10** [comment]
So I found it, it's the Vox Preamp MK2 - Supreme, Conqueror, Defiant; Virtuoso.

**Comment by cyrusadkisson** [comment]
According to voxshowroom.com, after Vox's first (not so great) foray into solid state, they adopted a modularized system for its next run at it. So I'm guessing that this GxVmk2.lv2 module is just the pre-amp circuitry of that modularization. 

http://www.voxshowroom.com/uk/amp/SSpreamp_hood.html

While this sounds amazing clean -- it really is a sweet tone you've created here -- there is no breakup whatsoever, even with the volume knob cranked all the way up. Is that intentional? I'm a DSP lay man, but is it possible to add the amplitude limiting components to this plugin to get distortion? Is it possible to add something like that as a separate plugin after this plugin in the chain?

**Comment by brummer10** [comment]
Yes, it's only the clean channel here. 
You should try it with the GxSuperFuzz in front https://github.com/brummer10/GxSuperFuzz.lv2
when you need some distortion. They come from the same time area.   
Alternative use the  GxSuppaToneBender https://github.com/brummer10/GxSuppaToneBender.lv2

**Comment by cyrusadkisson** [comment]
Ok so I love this vmk2 sim. It's probably the best clean plugin I've got, IMO. However, it stays completely clean at all times (which, I guess is to be expected). 

But I've got a tone I'm trying to dial that has _just a tiny, tiny bit_ of breakup in the clean tone. What plugin should I use to get that ever so slight sound of sizzling/breaking up without adding any additional character/EQ? (Most distortions have crazy EQ effects in addition to distortion.) 

(GxBoobTube gets pretty dang close, but I can't quite dial down the distortion far enough to simulate what I'm going for https://www.youtube.com/watch?v=NOG3eus4ZSo )

Edit: Nevermind. I was able to dial down GxBoobTube low enough to get just a little bit of dig/sparkle on the strings when digging in without overdoing it. GxBoobTube (being ultra transparent) might be my new favorite plug.

**Comment by brummer10** [comment]
You may also checkout the GxMultiBandDistortion, this one allow a very graduated saturation level.
